{"name": "Validating Output File against Input File", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'features/data/InstrumentDetails.csv'\n", "trace": "  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\output_file_validation.py\", line 9, in reading_input_files\n    input_df1 = pd.read_csv(\"features/data/InstrumentDetails.csv\")\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 912, in read_csv\n    return _read(filepath_or_buffer, kwds)\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 577, in _read\n    parser = TextFileReader(filepath_or_buffer, **kwds)\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 1407, in __init__\n    self._engine = self._make_engine(f, self.engine)\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 1661, in _make_engine\n    self.handles = get_handle(\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\common.py\", line 859, in get_handle\n    handle = open(\n"}, "steps": [{"name": "Given User reads InputFile1 InstrumentDetails.csv and InputFile2 PositionDetails.csv", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'features/data/InstrumentDetails.csv'\n", "trace": "  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\output_file_validation.py\", line 9, in reading_input_files\n    input_df1 = pd.read_csv(\"features/data/InstrumentDetails.csv\")\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 912, in read_csv\n    return _read(filepath_or_buffer, kwds)\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 577, in _read\n    parser = TextFileReader(filepath_or_buffer, **kwds)\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 1407, in __init__\n    self._engine = self._make_engine(f, self.engine)\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 1661, in _make_engine\n    self.handles = get_handle(\n  File \"d:\\learnings\\pycharmprojects\\pythonbehave\\venv\\lib\\site-packages\\pandas\\io\\common.py\", line 859, in get_handle\n    handle = open(\n"}, "start": 1681157083511, "stop": 1681157083648}, {"name": "When The user receives the OutputFile PositionReport.csv by application loading and transforming", "status": "skipped", "start": 1681157083650, "stop": 1681157083650}, {"name": "Then The user validates the Output File data against the InputFiles data", "status": "skipped", "start": 1681157083650, "stop": 1681157083650}], "start": 1681157083510, "stop": 1681157083650, "uuid": "4f98e98f-aab4-438c-bc41-e7f359341f64", "historyId": "73e2a4f364cf0e1937ebad434ac1ae3f", "fullName": "Verification of Output File: Validating Output File against Input File", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Verification of Output File"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}